docs/sources/installation/local.md

```diff
@@ -11,8 +11,23 @@ In order to log events with Grafana Loki, download and install both Promtail and
 
 The configuration specifies running Loki as a single binary.
 
-## Install
+## Install using APT or RPM package managers
```

Comment:
I think we want "manager" to be single, because you are using one or the other, not both at once.
```suggestion
## Install using APT or RPM package manager
```